akka-http-core/src/main/java/akka/http/javadsl/model/HttpHeader.java

```diff
@@ -26,7 +26,7 @@
     public abstract String lowercaseName();
 
     /**
-     * Returns true iff nameInLowerCase.equals(lowercaseName()).
+     * Returns true if nameInLowerCase.equals(lowercaseName()).
```

Comment:
I'm afraid this was not entirely a typo: "iff" is sometimes short for "if any only if" (https://en.wikipedia.org/wiki/If_and_only_if).

Might still be nice to replace this with something less obscure - perhaps "Returns true if nameInLowerCase.equals(lowercaseName()), false otherwise"? A bit verbose though.